src/client/search/TermItem.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+
+const TermItem = ({ onDeleteClick, onToggleParamTypeClick, query, source, paramTypes, id }) => (
+  <div id={`paramTypes${id}`} className="ui multiple compact dropdown labeled icon violet button"
+    onClick={(ee) => ee.stopPropagation()}
+  >
+    <span>
+      <i className={`${source} icon`}></i>
+      {query}
+    </span>
+    <i className="delete icon" onClick={() => {
+      onDeleteClick();
+    }}
+    ></i>
+    <TermItemMenu
+      termId={id}
+      highlightColor={source === 'twitter' ? 'blue' : ''}
+      paramTypes={paramTypes}
+      onToggleParamType={(paramTypeToggle) => {
+        onToggleParamTypeClick(paramTypeToggle);
+      }}
+    />
+  </div>
+);
+
+class TermItemMenu extends Component {
+  componentDidMount() {
+    $(`#paramTypes${this.props.termId}`)
```

Comment:
From above, you'd here search using this:
https://stackoverflow.com/questions/21756777/jquery-find-element-by-data-attribute-value

Feel free to keep it as an id rather than moving to data, I'd have gone for data.